src/NuGet.Core/NuGet.Build.Tasks/NuGet.targets

```diff
@@ -70,14 +70,20 @@ Copyright (c) .NET Foundation. All rights reserved.
   <PropertyGroup>
     <!-- Enable NuGetAudit by default -->
     <NuGetAudit Condition=" '$(NuGetAudit)' == '' ">true</NuGetAudit>
-    <!-- Report on low severity vulnerabilities, and higher. Allowed values are: low, moderate, high, critical -->
+    <!-- Report all severity vulnerabilities (low severity and higher). Allowed values are: low, moderate, high, critical -->
     <NuGetAuditLevel Condition=" '$(NuGetAuditLevel)' == '' ">low</NuGetAuditLevel>
-    <!-- Report known vulnerabilities on direct dependencies only. Allowed values are: direct, all -->
+    <!-- Report known vulnerabilities on direct and transitive dependencies, unless using .NET SDK with version lower than 9.0.100. Allowed values are: direct, all -->
     <NuGetAuditMode Condition="'$(NuGetAuditMode)' == ''
                     AND '$(NuGetExeSkipSdkAnalysisLevelCheck)' != 'true'
-                    AND $([MSBuild]::VersionGreaterThanOrEquals($([MSBuild]::ValueOrDefault('$(SdkAnalysisLevel)', '0.0')), '9.0.100'))"
-                    >all</NuGetAuditMode>
-    <NuGetAuditMode Condition=" '$(NuGetAuditMode)' == '' ">direct</NuGetAuditMode>
+                    AND '$(SdkAnalysisLevel)' != ''
+                    AND $([MSBuild]::VersionLessThan('$(SdkAnalysisLevel)', '9.0.100'))"
+                    >direct</NuGetAuditMode>
+    <NuGetAuditMode Condition="'$(NuGetAuditMode)' == ''
+                    AND '$(NuGetExeSkipSdkAnalysisLevelCheck)' != 'true'
+                    AND '$(UsingMicrosoftNETSdk)' == 'true'
```

Comment:
Looking at the references of [`UsingMicrosoftNETSdk`](https://github.com/search?q=repo%3ANuGet%2FNuGet.Client+UsingMicrosoftNETSdk&type=code) property in the repo, I see some casing differences. 

For example, the casing in the following line of code is different. My So `_project.GetPropertyValue("UsingMicrosoftNETSDK"); ` and `_project.GetPropertyValue("UsingMicrosoftNETSdk"); ` are functionally equivalent?

https://github.com/NuGet/NuGet.Client/blob/4a1faa2bcc771862a0ee1d241ebbfa1c196630ab/src/NuGet.Clients/NuGet.CommandLine/Commands/ProjectFactory.cs#L241

Reply:
MSBuild string comparisons are case insensitive: https://learn.microsoft.com/en-us/visualstudio/msbuild/msbuild-conditions?view=vs-2022